### PR TITLE
feat(analytics): adds referrers to issue links

### PR DESCRIPTION
**Pull request by scefali**
This PR adds a bunch of `referrer` query params to links heading to issues. The purpose of this is so we can understand the traffic to the issue page/stream better.

### CODE REVIEW

File: static/app/views/onboarding/setupDocs.tsx [hunk 1]
```diff
@@ -244,7 +244,9 @@ function SetupDocs({
       // If no projects remaining, then we can leave
       !project
     ) {
-      browserHistory.push('/');
+      browserHistory.push(
```
Comment (scefali) [inline static/app/views/onboarding/setupDocs.tsx:247]: Had to modify the URL so we can keep the referrer (it redirects to the issues list anyways)

File: static/app/components/events/userFeedback.tsx [hunk 2]
```diff
@@ -22,7 +22,7 @@ class EventUserFeedback extends Component<Props> {
   getUrl() {
     const {report, orgId, issueId} = this.props;
 
-    return `/organizations/${orgId}/issues/${issueId}/events/${report.eventID}/`;
+    return `/organizations/${orgId}/issues/${issueId}/events/${report.eventID}/?referrer=user_feedback`;
```
Comment (priscilawebdev) [inline static/app/components/events/userFeedback.tsx:25]: shall this be a dash? 
```suggestion
    return `/organizations/${orgId}/issues/${issueId}/events/${report.eventID}/?referrer=user-feedback`;
```

Comment (scefali) [reply static/app/components/events/userFeedback.tsx:25]: @priscilawebdev good catch!

File: static/app/components/eventOrGroupHeader.tsx [hunk 3]
```diff
@@ -116,6 +118,7 @@ function EventOrGroupHeader({
               eventID ? `events/${eventID}/` : ''
             }`,
             query: {
+              referrer: source || 'event-or-group-header',
```
Comment (priscilawebdev) [inline static/app/components/eventOrGroupHeader.tsx:121]: can this source be an empty `string` or `0`? ca we maybe only use `??` here?

Comment (scefali) [reply static/app/components/eventOrGroupHeader.tsx:121]: @priscilawebdev If someone put in an empty string or a 0, then defaulting to `'event-or-group-header` would be better

**Review by priscilawebdev — APPROVED** [review]
code looks good to me

**Review by scttcper — APPROVED** [review]

**Pull request merged**